password_security/controllers/main.py

```diff
@@ -1,39 +1,41 @@
+# pylint: disable=missing-docstring
 # Copyright 2015 LasLabs Inc.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 import operator
 
 from odoo import http
-from odoo.http import request
 from odoo.addons.auth_signup.controllers.main import AuthSignupHome
-from odoo.addons.web.controllers.main import ensure_db, Session
+from odoo.addons.web.controllers.main import Session, ensure_db
+from odoo.http import request
 
 from ..exceptions import PassError
 
 
+# pylint: disable=too-few-public-methods,protected-access
```

Comment:
Same here. What pylint config are you using? :thinking: 

See @OCA's configs in https://github.com/OCA/maintainer-quality-tools/tree/master/travis/cfg.